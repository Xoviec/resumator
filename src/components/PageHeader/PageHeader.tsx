import { Box, Button, Typography } from "@mui/material";
import { FC } from "react";
import { NavLink } from "react-router-dom";

type Props = {
  title: string;
};

export const PageHeader: FC<Props> = ({ title }) => {
  return (
    <Box display="flex" justifyContent="flex-start" alignItems="center">
      <Typography variant="h3" component="h1">
        {title} &nbsp;
      </Typography>
      <Button color="primary" variant="contained" component={NavLink} to="/">
        Go to overview
      </Button>
    </Box>
  );
};
