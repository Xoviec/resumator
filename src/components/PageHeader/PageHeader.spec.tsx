import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import PageHeader from "./PageHeader";

describe("PageHeader", () => {
  it("It renders correctly", () => {
    const intialRender = render(
      <BrowserRouter>
        <PageHeader title="test" />
      </BrowserRouter>
    );
    expect(intialRender).toMatchSnapshot();
  });
});
