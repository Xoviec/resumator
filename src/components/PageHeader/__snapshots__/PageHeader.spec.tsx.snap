// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader It renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-1k6yql2"
      >
        <h1
          class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
        >
          test
            
        </h1>
        <a
          aria-current="page"
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root active"
          href="/"
          tabindex="0"
        >
          Go to overview
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-1k6yql2"
    >
      <h1
        class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
      >
        test
          
      </h1>
      <a
        aria-current="page"
        class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root active"
        href="/"
        tabindex="0"
      >
        Go to overview
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
